src/plugins/intel_npu/src/al/include/intel_npu/icompiler.hpp

```diff
@@ -198,7 +49,7 @@ class ICompiler : public std::enable_shared_from_this<ICompiler> {
      *        including config options related to compilation
      * @return a shared pointer on an object implementing NetworkDescription interface
      */
-    virtual NetworkDescription compile(const std::shared_ptr<const ov::Model>& model, const Config& config) const = 0;
+    virtual NetworkDescription compile(const std::shared_ptr<const ov::Model>&, const Config&) const = 0;
```

Comment:
IMHO, it is a bit confusing to have names of the params listed in the brief comment, but not in the function definition.

Reply:
Yeah, they should be there, good catch

Reply:
done, thx